### DISCUSSION TITLE
connect DAX Studio to SSAS Tabular

### DISCUSSION THREAD

**Discussion by albertan352652** [DaxStudio/DaxStudio#600] [Q&A] [answered]
I opened a blank Power BI and connected to a dataset. I tried to use DAX studio to get tabular connection but I was not able to do that. In previous thread that was closed the suggestion was to connect DAX Studio to SSAS Tabular by way of finding a localhost tab number, i.e. by going to Power BI imported file and finding that under "Data Source Settings". But each time I click Data Source Settings it brings me to connection window to get connected to Dataset again. Can anyone please give some guidance on how can I connect Dax Studio to an imported Power BI file. Thanks!

**Comment by dgosbell** [comment]
I'm having trouble understanding your question. The title references SSAS Tabular, the the first sentence mentions a dataset, then finally you ask about an imported file. 

If you have Power BI Desktop open you can use the External Tools ribbon (as long as you installed DAX Studio using the All Users option) to launch DAX Studio and it will open connected to the data model from the file that you have open. Or we have a [guide here](https://daxstudio.org/tutorials/getting-connected/#pbidesktop) on how to connect to all the different supported data sources. If it's an imported file you just need to have the file open in Power BI Desktop and then just choose the "PBI / SSDT Model" option and choose your file from the dropdown.

**Reply by albertan352652** [reply]
I wanted to analyze the data in DAX studio from the Power BI file that I imported the data to from a dataset. In previous discussion under this link it says that the source could be found from Data Source Settings: https://github.com/DaxStudio/DaxStudio/issues/1

However I could not get connection to DAX studio because it is a Power BI file that is using imported data from a Dataset. When I click on Data Source Settings it brings me back to connection window with a Dataset again.

**Comment by dgosbell** [comment]
So that issue you are referencing is talking about a report using a Live Connection, not an imported dataset.

If you are using an imported dataset you would do the following:

1. Open your report in Power BI Desktop
2. Click on the **External Tools** ribbon
3. Click on the **DAX Studio** icon

**Reply by albertan352652** [reply]
Thank you. I am using an imported dataset. I downloaded the DAX studio but it only appears on my Excel as "Add in". It did not appear on Power BI desktop under External Tools (I do have ALM Toolkit, Analyze in Excel and Tabular Editor there but not DAX). When I use DAX studio from Excel it doesn't get connected to Power BI that uses an imported dataset. I will try to uninstall DAX studio and install it again I think...

**Reply by dgosbell** [reply, marked as answer]
That could happen if you originally chose the **Current User** option when installing instead of the default **All Users** option. If you uninstall then re-install choosing the **All Users** option the external tools option should be enabled.

But you should also be able to do the following:

1. Open your report in Power BI Desktop
2. Open DAX Studio and choose the **PBI / SSDT Model** option and make sure your report is selected in the dropdown
3. Click **Connect**
